code/modules/surgery/organs/internal/ears/_ears.dm

```diff
@@ -225,3 +236,46 @@
 	if(. & EMP_PROTECT_SELF)
 		return
 	apply_organ_damage(20 / severity)
+
+// Overlay applied for the eccentric ear types
+/datum/bodypart_overlay/mutant/ears
+	layers = EXTERNAL_FRONT|EXTERNAL_BEHIND
+	color_source = ORGAN_COLOR_HAIR
+	feature_key = "ears"
+
+/datum/bodypart_overlay/mutant/ears/get_global_feature_list()
+	return SSaccessories.ears_list
+
+/datum/bodypart_overlay/mutant/ears/can_draw_on_bodypart(mob/living/carbon/human/human)
+	var/obj/item/bodypart/head/head = human.get_bodypart(BODY_ZONE_HEAD)
+	if(isnull(head) || IS_ROBOTIC_LIMB(head))
+		return FALSE
+	for(var/obj/item/worn_item in human.get_equipped_items())
+		if(worn_item.flags_inv & HIDEHAIR)
+			return FALSE
+	return TRUE
+
+/datum/bodypart_overlay/mutant/ears/get_image(image_layer, obj/item/bodypart/limb)
+	var/mutable_appearance/created = ..()
+	var/mutable_appearance/inner = mutable_appearance(sprite_datum.icon, layer = image_layer, appearance_flags = RESET_COLOR)
+	var/gender = (sprite_datum.gender_specific && limb?.limb_gender == FEMALE) ? "f" : "m"
+	inner.icon_state = "[gender]_[feature_key]inner_[sprite_datum.icon_state]_[mutant_bodyparts_layertext(image_layer)]"
+
+	if(sprite_datum.center)
+		center_image(inner, sprite_datum.dimension_x, sprite_datum.dimension_y)
+
+	created.underlays += inner
+	return created
+
+/datum/bodypart_overlay/mutant/ears/felinid
+	dna_block = DNA_EARS_BLOCK
+
+/datum/bodypart_overlay/mutant/ears/felinid/New()
+	. = ..()
+	set_appearance(/datum/sprite_accessory/ears/cat)
+
+/datum/bodypart_overlay/mutant/ears/fox
+
+/datum/bodypart_overlay/mutant/ears/fox/New()
+	. = ..()
+	set_appearance(/datum/sprite_accessory/ears/fox)
```

Comment:
You can set the sprite_datum var and set imprint_on_next_insertion to FALSE for an override like this